source/time_tables/tests/get_stop_times_test.cpp

```diff
@@ -196,15 +201,17 @@ BOOST_AUTO_TEST_CASE(test_frequency_for_calendar) {
     ed::builder b("20120614");
     std::string spa1 = "stop1";
     DateTime vj1_departure = 8000;
-    DateTime vj2_departure = 8001;
+    //DateTime vj2_departure = 8001;
     size_t headway_sec = 100;
     b.frequency_vj("A", vj1_departure, 9000, headway_sec,
                    "default_network", "1010", "", true, "vj1")
                     (spa1, vj1_departure, vj1_departure)
-                    ("useless_stop", 1000, 1000);
+                    ("useless_stop", 10000, 10000);
+    /*
     b.vj("A", "1010", "", true, "vj2")
         (spa1, vj2_departure, vj2_departure)
-        ("useless_stop", 1000, 1000);
+        ("useless_stop", 10001, 10001);
+    */
```

Comment:
The trouble here is that our journey patterns do not mix frequency and discrete vj because testing if they overlap is difficult. But this test check that. Maybe testing that at a higher level (as a departure board of a route that have freq and discrete vjs).